bom/pom.xml

```diff
@@ -98,8 +98,8 @@
 
 
     <properties>
-        <antlr4.version>4.9.3</antlr4.version>
-        <archie.version>2.0.1</archie.version>
+        <antlr4.version>4.11.1</antlr4.version>
```

Comment:
Is there a reason why we have to archie and antlr4 versions separately from the SDK?